app/cli.py

```diff
@@ -7,7 +7,7 @@
 from .models import Category, Language, Resource
 
 
-def import_resources(db):
+def import_resources(db):   # pragma: no cover
```

Comment:
This is how you exclude functions for the test coverage

Reply:
LG2M